aimsprop/traj.py

```diff
@@ -65,9 +64,21 @@ def __lt__(
         """
         return (self.t, self.label, self.I) < (other.t, other.label, other.I)
 
+    def __str__(self):
+        return f"""label: {self.label}
+        time: {self.t}
+        weight: {self.w}
+        state: {self.I}
+        atomic number: {self.N}
+        xyz: {self.xyz}
+        properties: {self.properties}
+        """
 
-class Trajectory(object):
+    def _repr_html_(self):
+        return "<p>" + str(self).replace("\n", "<br>") + "</p>"
 
+
+class Trajectory(object):
```

Comment:
This all looks good! You may want to add a very simple test where you instantiate a frame and assert that the string method returns a str. These little things help ensure code is always functional and we don't ship a version with something trivial broken--liek a bad __str__ method--that breaks everyone's scripts :)

```
def test_frame___str__():
    frame = Frame(...)
    assert isinsntance(str(frame), str)
```